### PR TITLE
New package: GeorgieLabs.SoundWireServer version 2.5.0

### PR DESCRIPTION
**Pull request by matracey**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

### CODE REVIEW

**Comment by stephengillie** [comment]
image

- https://github.com/microsoft/winget-pkgs/issues/107639

Verify: 
image

File: manifests/g/GeorgieLabs/SoundWireServer/2.5.0/GeorgieLabs.SoundWireServer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: GeorgieLabs.SoundWireServer
+PackageVersion: 2.5.0
```
Comment (stephengillie) [inline manifests/g/GeorgieLabs/SoundWireServer/2.5.0/GeorgieLabs.SoundWireServer.installer.yaml:5]:
```suggestion
PackageVersion: "2.5"
```
Please update file name and path to match this change.

File: manifests/g/GeorgieLabs/SoundWireServer/2.5.0/GeorgieLabs.SoundWireServer.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: GeorgieLabs.SoundWireServer
+PackageVersion: 2.5.0
```
Comment (stephengillie) [inline manifests/g/GeorgieLabs/SoundWireServer/2.5.0/GeorgieLabs.SoundWireServer.yaml:5]:
```suggestion
PackageVersion: "2.5"
```
Please update file name and path to match this change.

File: manifests/g/GeorgieLabs/SoundWireServer/2.5.0/GeorgieLabs.SoundWireServer.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: GeorgieLabs.SoundWireServer
+PackageVersion: 2.5.0
```
Comment (stephengillie) [inline manifests/g/GeorgieLabs/SoundWireServer/2.5.0/GeorgieLabs.SoundWireServer.locale.en-US.yaml:5]:
```suggestion
PackageVersion: "2.5"
```
Please update file name and path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @matracey,

The `PackageVersion` should match the Registry when possible. Is this package marketed with `2.5.0` as the version? If so, a `DisplayVersion` might also help.

File: manifests/g/GeorgieLabs/SoundWireServer/2.5.0/GeorgieLabs.SoundWireServer.installer.yaml (hunk 1 above)
Comment (matracey) [reply manifests/g/GeorgieLabs/SoundWireServer/2.5.0/GeorgieLabs.SoundWireServer.installer.yaml]: Thanks! I've made the change to all manifest files and updated the path too.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**